python/src/opendp/measurements.py

```diff
@@ -1098,10 +1164,18 @@ def then_report_noisy_max_gumbel(
     scale: Any,
     optimize: str,
     QO: Optional[RuntimeTypeDescriptor] = None
-):
-    r"""Partial constructructor which uses the preceding output domain and metric as its input domain and metric. See also:
+):  
+    r"""partial constructor of make_report_noisy_max_gumbel
 
-    :py:func:`opendp.measurements.make_report_noisy_max_gumbel`
+    .. seealso:: 
+      Delays application of `input_domain` and `input_metric` in :py:func:`opendp.measurements.make_report_noisy_max_gumbel`
+
+    :param scale: Higher scales are more private.
+    :type scale: Any
+    :param optimize: Indicate whether to privately return the "Max" or "Min"
```

Comment:
I'm not sure this is in sync with the rust, or at least there seems to be some upper/lowercase inconsistency:
```
    let optimize = match try_!(to_str(optimize)) {
        i if i.to_lowercase().starts_with("min") => Optimize::Min,
        i if i.to_lowercase().starts_with("max") => Optimize::Max,
        _ => return err!(FFI, "optimize must start with \"min\" or \"max\"").into(),
    };
```
... but I wonder if we should even have string arguments here: A boolean or enum seems more natural. Probably not something to block this PR: Suggest filing a new issue from this comment.